mycroft/audio/__main__.py

```diff
@@ -40,6 +41,13 @@ def main():
     audio = AudioService(bus)  # Connect audio service instance to message bus
     create_daemon(bus.run_forever)
 
+    try:
+        LOG.info("Getting Pre-loaded cache")
+        cache_handler.main()
```

Comment:
Since this only applies to Mimic2 maybe this should be moved to be called as the mimic2 tts is being created?